app/models/student.rb

```diff
@@ -0,0 +1,5 @@
+class Student < ActiveRecord::Base
+  validates :first_name, :last_name, :email, :cell_phone, :home_phone, :work_phone, presence: true
```

Comment:
With this many fields being validated, it makes it easier to read what the validation is by using the `validates_presence_of` method. But otherwise looks good!